frontend/src/pages/profile/components/user-articles-genres-stats/libs/components/genres-list/genres-list.tsx

```diff
@@ -0,0 +1,28 @@
+import { type UserArticlesGenreStatsItem } from '~/packages/users/users.js';
+
+import { GENRES_CHART_COLORS } from '../../constants/constants.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+  chartData: UserArticlesGenreStatsItem[];
```

Comment:
B18

Reply:
But the word "data" is already plural form

Reply:
but the word "data" already has a plural form

Reply:
Maybe let's call this `genresStatistics` - just to make it more precise what does it mean